[2019-03-27T22:22:40.642] [ERROR] measure - 参数错误 Error: 参数错误
    at updateData (/Users/yp-tc-m2504/Documents/f2e/amorphous_node/controller/measure.js:211:15)
    at Layer.handle [as handle_request] (/Users/yp-tc-m2504/Documents/f2e/amorphous_node/node_modules/express/lib/router/layer.js:95:5)
    at next (/Users/yp-tc-m2504/Documents/f2e/amorphous_node/node_modules/express/lib/router/route.js:137:13)
    at Route.dispatch (/Users/yp-tc-m2504/Documents/f2e/amorphous_node/node_modules/express/lib/router/route.js:112:3)
    at Layer.handle [as handle_request] (/Users/yp-tc-m2504/Documents/f2e/amorphous_node/node_modules/express/lib/router/layer.js:95:5)
    at /Users/yp-tc-m2504/Documents/f2e/amorphous_node/node_modules/express/lib/router/index.js:281:22
    at Function.process_params (/Users/yp-tc-m2504/Documents/f2e/amorphous_node/node_modules/express/lib/router/index.js:335:12)
    at next (/Users/yp-tc-m2504/Documents/f2e/amorphous_node/node_modules/express/lib/router/index.js:275:10)
    at Function.handle (/Users/yp-tc-m2504/Documents/f2e/amorphous_node/node_modules/express/lib/router/index.js:174:3)
    at router (/Users/yp-tc-m2504/Documents/f2e/amorphous_node/node_modules/express/lib/router/index.js:47:12)
    at Layer.handle [as handle_request] (/Users/yp-tc-m2504/Documents/f2e/amorphous_node/node_modules/express/lib/router/layer.js:95:5)
    at trim_prefix (/Users/yp-tc-m2504/Documents/f2e/amorphous_node/node_modules/express/lib/router/index.js:317:13)
    at /Users/yp-tc-m2504/Documents/f2e/amorphous_node/node_modules/express/lib/router/index.js:284:7
    at Function.process_params (/Users/yp-tc-m2504/Documents/f2e/amorphous_node/node_modules/express/lib/router/index.js:335:12)
    at next (/Users/yp-tc-m2504/Documents/f2e/amorphous_node/node_modules/express/lib/router/index.js:275:10)
    at checkLogin (/Users/yp-tc-m2504/Documents/f2e/amorphous_node/routes/index.js:23:5)
[2019-03-27T22:24:14.336] [ERROR] measure - 参数错误 Error: 参数错误
    at updateData (/Users/yp-tc-m2504/Documents/f2e/amorphous_node/controller/measure.js:211:15)
    at Layer.handle [as handle_request] (/Users/yp-tc-m2504/Documents/f2e/amorphous_node/node_modules/express/lib/router/layer.js:95:5)
    at next (/Users/yp-tc-m2504/Documents/f2e/amorphous_node/node_modules/express/lib/router/route.js:137:13)
    at Route.dispatch (/Users/yp-tc-m2504/Documents/f2e/amorphous_node/node_modules/express/lib/router/route.js:112:3)
    at Layer.handle [as handle_request] (/Users/yp-tc-m2504/Documents/f2e/amorphous_node/node_modules/express/lib/router/layer.js:95:5)
    at /Users/yp-tc-m2504/Documents/f2e/amorphous_node/node_modules/express/lib/router/index.js:281:22
    at Function.process_params (/Users/yp-tc-m2504/Documents/f2e/amorphous_node/node_modules/express/lib/router/index.js:335:12)
    at next (/Users/yp-tc-m2504/Documents/f2e/amorphous_node/node_modules/express/lib/router/index.js:275:10)
    at Function.handle (/Users/yp-tc-m2504/Documents/f2e/amorphous_node/node_modules/express/lib/router/index.js:174:3)
    at router (/Users/yp-tc-m2504/Documents/f2e/amorphous_node/node_modules/express/lib/router/index.js:47:12)
    at Layer.handle [as handle_request] (/Users/yp-tc-m2504/Documents/f2e/amorphous_node/node_modules/express/lib/router/layer.js:95:5)
    at trim_prefix (/Users/yp-tc-m2504/Documents/f2e/amorphous_node/node_modules/express/lib/router/index.js:317:13)
    at /Users/yp-tc-m2504/Documents/f2e/amorphous_node/node_modules/express/lib/router/index.js:284:7
    at Function.process_params (/Users/yp-tc-m2504/Documents/f2e/amorphous_node/node_modules/express/lib/router/index.js:335:12)
    at next (/Users/yp-tc-m2504/Documents/f2e/amorphous_node/node_modules/express/lib/router/index.js:275:10)
    at checkLogin (/Users/yp-tc-m2504/Documents/f2e/amorphous_node/routes/index.js:23:5)
[2019-03-27T22:28:48.638] [ERROR] measure - 参数错误 Error: 参数错误
    at updateData (/Users/yp-tc-m2504/Documents/f2e/amorphous_node/controller/measure.js:211:15)
    at Layer.handle [as handle_request] (/Users/yp-tc-m2504/Documents/f2e/amorphous_node/node_modules/express/lib/router/layer.js:95:5)
    at next (/Users/yp-tc-m2504/Documents/f2e/amorphous_node/node_modules/express/lib/router/route.js:137:13)
    at Route.dispatch (/Users/yp-tc-m2504/Documents/f2e/amorphous_node/node_modules/express/lib/router/route.js:112:3)
    at Layer.handle [as handle_request] (/Users/yp-tc-m2504/Documents/f2e/amorphous_node/node_modules/express/lib/router/layer.js:95:5)
    at /Users/yp-tc-m2504/Documents/f2e/amorphous_node/node_modules/express/lib/router/index.js:281:22
    at Function.process_params (/Users/yp-tc-m2504/Documents/f2e/amorphous_node/node_modules/express/lib/router/index.js:335:12)
    at next (/Users/yp-tc-m2504/Documents/f2e/amorphous_node/node_modules/express/lib/router/index.js:275:10)
    at Function.handle (/Users/yp-tc-m2504/Documents/f2e/amorphous_node/node_modules/express/lib/router/index.js:174:3)
    at router (/Users/yp-tc-m2504/Documents/f2e/amorphous_node/node_modules/express/lib/router/index.js:47:12)
    at Layer.handle [as handle_request] (/Users/yp-tc-m2504/Documents/f2e/amorphous_node/node_modules/express/lib/router/layer.js:95:5)
    at trim_prefix (/Users/yp-tc-m2504/Documents/f2e/amorphous_node/node_modules/express/lib/router/index.js:317:13)
    at /Users/yp-tc-m2504/Documents/f2e/amorphous_node/node_modules/express/lib/router/index.js:284:7
    at Function.process_params (/Users/yp-tc-m2504/Documents/f2e/amorphous_node/node_modules/express/lib/router/index.js:335:12)
    at next (/Users/yp-tc-m2504/Documents/f2e/amorphous_node/node_modules/express/lib/router/index.js:275:10)
    at checkLogin (/Users/yp-tc-m2504/Documents/f2e/amorphous_node/routes/index.js:23:5)
[2019-03-27T22:30:37.246] [ERROR] measure - 参数错误 Error: 参数错误
    at updateData (/Users/yp-tc-m2504/Documents/f2e/amorphous_node/controller/measure.js:211:15)
    at Layer.handle [as handle_request] (/Users/yp-tc-m2504/Documents/f2e/amorphous_node/node_modules/express/lib/router/layer.js:95:5)
    at next (/Users/yp-tc-m2504/Documents/f2e/amorphous_node/node_modules/express/lib/router/route.js:137:13)
    at Route.dispatch (/Users/yp-tc-m2504/Documents/f2e/amorphous_node/node_modules/express/lib/router/route.js:112:3)
    at Layer.handle [as handle_request] (/Users/yp-tc-m2504/Documents/f2e/amorphous_node/node_modules/express/lib/router/layer.js:95:5)
    at /Users/yp-tc-m2504/Documents/f2e/amorphous_node/node_modules/express/lib/router/index.js:281:22
    at Function.process_params (/Users/yp-tc-m2504/Documents/f2e/amorphous_node/node_modules/express/lib/router/index.js:335:12)
    at next (/Users/yp-tc-m2504/Documents/f2e/amorphous_node/node_modules/express/lib/router/index.js:275:10)
    at Function.handle (/Users/yp-tc-m2504/Documents/f2e/amorphous_node/node_modules/express/lib/router/index.js:174:3)
    at router (/Users/yp-tc-m2504/Documents/f2e/amorphous_node/node_modules/express/lib/router/index.js:47:12)
    at Layer.handle [as handle_request] (/Users/yp-tc-m2504/Documents/f2e/amorphous_node/node_modules/express/lib/router/layer.js:95:5)
    at trim_prefix (/Users/yp-tc-m2504/Documents/f2e/amorphous_node/node_modules/express/lib/router/index.js:317:13)
    at /Users/yp-tc-m2504/Documents/f2e/amorphous_node/node_modules/express/lib/router/index.js:284:7
    at Function.process_params (/Users/yp-tc-m2504/Documents/f2e/amorphous_node/node_modules/express/lib/router/index.js:335:12)
    at next (/Users/yp-tc-m2504/Documents/f2e/amorphous_node/node_modules/express/lib/router/index.js:275:10)
    at checkLogin (/Users/yp-tc-m2504/Documents/f2e/amorphous_node/routes/index.js:23:5)
[2019-03-27T22:35:26.726] [ERROR] measure - 更新数据失败 Error: 更新数据失败
    at MeasureService.allOutStore (/Users/yp-tc-m2504/Documents/f2e/amorphous_node/service/measure.js:85:15)
    at <anonymous>
    at process._tickCallback (internal/process/next_tick.js:188:7)
[2019-03-27T22:37:48.878] [ERROR] measure - 更新数据失败 Error: 更新数据失败
    at MeasureService.allOutStore (/Users/yp-tc-m2504/Documents/f2e/amorphous_node/service/measure.js:85:15)
    at <anonymous>
    at process._tickCallback (internal/process/next_tick.js:188:7)
[2019-03-27T22:38:49.825] [ERROR] measure - 更新数据失败 Error: 更新数据失败
    at MeasureService.allOutStore (/Users/yp-tc-m2504/Documents/f2e/amorphous_node/service/measure.js:85:15)
    at <anonymous>
    at process._tickCallback (internal/process/next_tick.js:188:7)
[2019-03-27T22:39:56.182] [ERROR] measure - 更新数据失败 Error: 更新数据失败
    at MeasureService.allOutStore (/Users/yp-tc-m2504/Documents/f2e/amorphous_node/service/measure.js:85:15)
    at <anonymous>
    at process._tickCallback (internal/process/next_tick.js:188:7)
[2019-03-27T22:40:27.461] [ERROR] measure - 更新数据失败 Error: 更新数据失败
    at MeasureService.allOutStore (/Users/yp-tc-m2504/Documents/f2e/amorphous_node/service/measure.js:85:15)
    at <anonymous>
    at process._tickCallback (internal/process/next_tick.js:188:7)
[2019-03-27T22:45:00.720] [ERROR] measure - 更新数据失败 Error: 更新数据失败
    at MeasureService.allOutStore (/Users/yp-tc-m2504/Documents/f2e/amorphous_node/service/measure.js:85:15)
    at <anonymous>
    at process._tickCallback (internal/process/next_tick.js:188:7)
[2019-03-27T22:46:33.090] [ERROR] measure - 更新数据失败 Error: 更新数据失败
    at MeasureService.allOutStore (/Users/yp-tc-m2504/Documents/f2e/amorphous_node/service/measure.js:85:15)
    at <anonymous>
    at process._tickCallback (internal/process/next_tick.js:188:7)
[2019-03-28T23:13:24.454] [ERROR] measure - 添加仓位失败：某些炉号或盘号不存在
[2019-03-28T23:14:20.370] [ERROR] measure - 添加仓位失败：某些炉号或盘号不存在
[2019-03-28T23:14:40.241] [ERROR] measure - 添加仓位失败：某些炉号或盘号不存在
[2019-03-28T23:15:44.814] [ERROR] measure - 添加仓位失败：某些炉号或盘号不存在
[2019-03-28T23:21:31.389] [ERROR] measure - Cast to number failed for value "NaN" at path "coilNumber" for model "Measure"
[2019-03-28T23:24:33.046] [ERROR] measure - Cast to number failed for value "NaN" at path "coilNumber" for model "Measure"
[2019-03-28T23:25:48.805] [ERROR] measure - Cast to number failed for value "NaN" at path "coilNumber" for model "Measure"
[2019-03-29T11:10:28.908] [ERROR] measure - Cannot read property 'data' of undefined
[2019-03-29T11:10:48.034] [ERROR] measure - Cannot read property 'data' of undefined
[2019-03-29T11:25:27.452] [ERROR] measure - item is not defined
[2019-03-29T11:26:35.645] [ERROR] measure - Cannot read property 'furnace' of undefined
[2019-03-29T11:29:36.113] [ERROR] measure - Cannot read property 'furnace' of undefined
[2019-03-29T15:14:13.877] [ERROR] measure - 添加仓位失败：某些炉号或盘号不存在
[2019-03-29T15:24:35.037] [ERROR] measure - furnace is not defined
[2019-03-29T15:24:58.153] [ERROR] measure - 添加仓位失败：炉号06-20190214-02/01，盘号4不存在
[2019-03-29T15:25:53.805] [ERROR] measure - 添加仓位失败：炉号06-20190214-02/01，盘号4不存在
[2019-03-29T15:47:41.156] [ERROR] measure - 添加仓位失败
[2019-03-29T15:48:04.149] [ERROR] measure - 添加仓位失败
[2019-03-29T15:48:42.945] [ERROR] measure - 添加仓位失败
[2019-03-29T15:50:32.958] [ERROR] measure - 添加仓位失败
[2019-03-29T15:50:51.721] [ERROR] measure - 添加仓位失败
[2019-03-29T15:51:09.612] [ERROR] measure - 添加仓位失败
[2019-03-29T15:51:46.446] [ERROR] measure - 添加仓位失败
[2019-03-29T15:52:19.372] [ERROR] measure - 添加仓位失败
[2019-03-29T15:52:52.251] [ERROR] measure - 添加仓位失败
[2019-03-29T15:53:20.507] [ERROR] measure - 添加仓位失败
[2019-03-29T15:54:34.864] [ERROR] measure - 添加仓位失败
